en/02_Developer_Guides/03_Forms/01_Validation.md

```diff
@@ -75,58 +98,28 @@ public function validate($validator)
 {
     if ((int) $this->Value() === 10) {
         $validator->validationError($this->Name(), 'This value cannot be 10');
-        return false;
+        return $this->extendValidationResult(false, $validator);
     }
 
-    return true;
+    return $this->extendValidationResult(true, $validator);
 }
 ```
 
-The `validate` method should return `true` if the value passes any validation and `false` if Silverstripe CMS should trigger
-a validation error on the page. In addition a useful error message must be set on the given validator.
+The `validate` method should compute a boolean (`true` if the value passes validation and `false` if Silverstripe CMS 
+should trigger a validation error on the page) and pass this to the `extendValidationResult` method to allow extensions
+to hook into the validation logic. In addition, in the event of failed validation, a useful error message must be set
+on the given validator.
 
 [notice]
 You can also override the entire `Form` validation by subclassing `Form` and defining a `validate` method on the form.
 [/notice]
 
-Say we need a custom `FormField` which requires the user input a value in a `TextField` between 2 and 5. There would be
-two ways to go about this:
-
-A custom `FormField` which handles the validation. This means the `FormField` can be reused throughout the site and have
-the same validation logic applied to it throughout.
-
-**app/src/CustomNumberField.php**
-
-```php
-use SilverStripe\Forms\TextField;
-
-class CustomNumberField extends TextField
```

Comment:
This section and example is just repeating the section above (the “this value cannot be 10” example)